lib/cmd/ls.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const path = require('path');
+const Command = require('../command');
+const isWin = process.platform === 'win32';
+const osRelated = {
+  titlePrefix: isWin ? '\\"title\\":\\"' : '"title":"',
+  appWorkerPath: isWin ? 'egg-cluster\\lib\\app_worker.js' : 'egg-cluster/lib/app_worker.js',
+  agentWorkerPath: isWin ? 'egg-cluster\\lib\\agent_worker.js' : 'egg-cluster/lib/agent_worker.js',
+};
+
+class LsCommand extends Command {
+
+  constructor(rawArgv) {
+    super(rawArgv);
+    this.usage = 'Usage: egg-scripts ls [--title=example]';
+    this.serverBin = path.join(__dirname, '../start-cluster');
+    this.options = {
+      title: {
+        description: 'process title description, use for find app',
+        type: 'string',
+      },
+    };
+  }
+
+  get description() {
+    return 'ls app';
+  }
+
+  * run(context) {
+    const { argv } = context;
+    this.logger.info(`list egg application ${argv.title ? `with --title=${argv.title}` : ''}`);
+
+    const processList = yield this.helper.findNodeProcessWithPpid(item => {
```

Comment:
You can move the filter into findNodeProcessWithPpid.

Reply:
@popomore This filter function use a title variable , so I think it's more appropriate to put it here.

Reply:
maybe this could extract to a utils function, and share with stop cmd